<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibClient</class>
 <widget class="QMainWindow" name="LibClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QGridLayout" name="BookFilter">
      <item row="0" column="0">
       <widget class="QGroupBox" name="Filters">
        <property name="title">
         <string>Filters:</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="InputFilter">
           <item>
            <widget class="QLabel" name="labelTitle">
             <property name="text">
              <string>Title:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fTitle"/>
           </item>
           <item>
            <widget class="QLabel" name="labelAuthor">
             <property name="text">
              <string>Author: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="fAuthor"/>
           </item>
           <item>
            <widget class="QLabel" name="labelYear">
             <property name="text">
              <string>Year: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="fYear">
             <property name="contextMenuPolicy">
              <enum>Qt::DefaultContextMenu</enum>
             </property>
             <property name="toolTipDuration">
              <number>-1</number>
             </property>
             <property name="specialValueText">
              <string>Any</string>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>2100</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="displayIntegerBase">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bClear">
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="BookAction">
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="FunButtons">
        <item>
         <widget class="QPushButton" name="bAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bDelete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableBooks"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
